lib/patches/before/openssl.rb

```diff
@@ -0,0 +1 @@
+require "openssl-stubs" unless ENV['TRUFFLERUBY_OPENSSL_ENABLED']
```

Comment:
Alternatively it could be:
`require "openssl-stubs" if ENV['TRUFFLERUBY_OPENSSL_DISABLED']`